src/pages/Auth/SignIn/style.ts

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components'
+import {Form, Button} from 'antd'
+
+export const SignInContainer = styled.div`
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+`
+
+export const SignInForm = styled(Form)`
+	width: 35rem;
```

Comment:
I think `max-width` would be a better for this? As the layout will break if, says user is using an iPhone 5S with 320px  width

Reply:
No worries, since we use rem (not px). I have tested it against all devices